modules/payments_stripe/public/lib/commands/stripe_checkout/complete/map_request.liquid

```diff
@@ -6,6 +6,10 @@
 
     hash_assign data['valid'] = true
     hash_assign data['external_id'] = response.client_reference_id
+    function transaction = 'modules/payments/queries/transactions/find', id: data.external_id
+    hash_assign data['transaction'] = transaction
+    assign customer = nil | hash_merge: gateway_id: response.customer, email: response.customer_details.email, name: response.customer_details.name, user_id: transaction.user_id
```

Comment:
@streflik You should probably assign `response.customer` to customer_id. 
`gateway_id` seems obsolete as it is always in the scope of `stripe` payment gateway.